api/src/org/apache/cloudstack/api/command/admin/domain/ListDomainsCmd.java

```diff
@@ -53,6 +59,12 @@
                description = "If set to false, list only resources belonging to the command's caller; if set to true - list resources that the caller is authorized to see. Default value is false")
     private Boolean listAll;
 
+    @Parameter(name = ApiConstants.DETAILS,
+               type = CommandType.LIST,
+               collectionType = CommandType.STRING,
+               description = "comma separated list of host details requested, value can be a list of [ all, resource, min]")
```

Comment:
@serg38 currently no difference. However if we want to add more type of details, then it will be useful. for example, we add a new detail type: stats in our branch.

the defition is copied from listhosts API (you can see host details in the decription, I will change it).